.gitattributes

```
@@ -1,2 +1,4 @@
-/*/**/Dockerfile       linguist-generated
-/Dockerfile*.template  linguist-language=Dockerfile
+/*/**/Dockerfile       linguist-generated eol=lf
+/Dockerfile*.template  linguist-language=Dockerfile eol=lf
+*.sh                   eol=lf
+*.awk                  eol=lf
```

I was hoping we could avoid being quite this explicit, but this is fair -- I think we can simplify by generalizing to _all_ text files, though:

```suggestion
*                      text=auto eol=lf
/*/**/Dockerfile       linguist-generated
/Dockerfile*.template  linguist-language=Dockerfile
```

(If I'm reading https://git-scm.com/docs/gitattributes correctly)

---

Thanks, done.